docs/user-guide/persistent-volumes/index.md

```diff
@@ -193,13 +192,14 @@ In the CLI, the access modes are abbreviated to:
 ### Class
 
 A PV can have a class, which is specified by setting the
-`volume.beta.kubernetes.io/storage-class` annotation to the name of a
+`storageClassName` attribute to the name of a
 `StorageClass`. A PV of a particular class can only be bound to PVCs requesting
-that class. A PV with no annotation or its class annotation set to `""` has no
-class and can only be bound to PVCs that request no particular class.
+that class. A PV with no `storageClassName` has no class and can only be bound
+to PVCs that request no particular class.
 
-In the future after beta, the `volume.beta.kubernetes.io/storage-class`
-annotation will become an attribute.
+In the past, annotation `volume.beta.kubernetes.io/storage-class` was used instead
```

Comment:
Grammar -- add 'the' before 'annotation': "In the past, **the** annotation `volume.beta.kubernetes.io/storage-class`"